packages/sources/index.js

```diff
@@ -70,8 +72,10 @@ const getRouteBySource = (src) => routeBySource[src];
 
 // code-du-travail -> code_du_travail
 /** @type {cdtnSources.getSourceByRouteFn} */
-const getSourceByRoute = (slug) =>
-  sources.find((key) => routeBySource[key] === slug) || slug;
+const getSourceByRoute = (slug) => {
+  const [source] = routes.find(([, value]) => slug === value) || [];
+  return source ? source : null;
```

Comment:
If think `source | null`is enough here but not sure

Reply:
if you mean `return source || null` ?   it's the same

Reply:
👌